app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -2,6 +2,11 @@
 
 module ExceptionHunter
   class ErrorsController < ApplicationController
+    helper_method :dashboard_user
+    before_action do
+      send("authenticate_#{dashboard_user}!") if dashboard_user
```

Comment:
We should hide this inside our domain, dark magic (metaprogramming) should be well encapsulated.